definitions/features/capsule.rb

```diff
@@ -0,0 +1,15 @@
+require 'features/foreman_proxy'
+
+class Features::Capsule < Features::ForemanProxy
+  metadata do
+    label :capsule
+
+    confine do
+      smart_proxy? && feature(:installer) && feature(:installer).last_scenario.eql?('capsule')
```

Comment:
This confine won't work for installer without scenarios (<=6.2). We should confirm if we still want to support those.

Reply:
The `smart_proxy?` limits this to external capsules only. Is this intentional? IMO we should get rid of `smart_proxy?` helper completely once this is covered by the features. We don't want to have duplicity in this logic. When we inherit the from the feature can we call the 'super' confine?

Reply:
@mbacovsky
Sure, thank you for pointing it.
I will confirm if we still want to support capsule scenarios for (<= 6.2).

Verified that `smart_proxy?` method not used anywhere so we can remove this completely. 
One query - do you want me to modify behavior of `capsule` in such way which will handle both external and in-built proxy in case of downstream?
I thought for external capsule we have different upgrade as well as installation setup than in-built proxy which is why I kept foreman-proxy feature as it is and inherited it.